doc/rules/language_construct/silenced_deprecation_error.rst

```diff
@@ -0,0 +1,27 @@
+===================================
+Rule ``silenced_deprecation_error``
+===================================
```

Comment:
I don't know if GitHub is just rendering this wrong, but when looking at it [(link for convenience)](https://github.com/julienfalque/PHP-CS-Fixer/blob/doc/doc/rules/language_construct/silenced_deprecation_error.rst), it's not really styled and thus just a blob of text where it's unsure what the first warning is, and where the summary starts.

I'd say do the summary and description either before or after the warnings, but not in between. (And the styling should be fixed.)

Reply:
I would keep the current ordering:
- deprecation notice (if deprecated)
- summary
- description (if available)
- risk details (if risky)

The main issue is that Github has no specific styling for admonitions such as `.. warning::`, not sure what we can do about it.

Reply:
I know you won't like this answer, but manual styling is always an option:

**Warning:** deprecated, see `other_rule`

This fixer does X

*Risky: because Y*

😜

Reply:
Indeed I don't like this solution: it would work for Github but it means we would not use dedicated markup provided by reStructuredText and thus no possibility to leverage it for rendering the documentation somewhere else such as e.g. at https://cs.symfony.com.

Reply:
I get what you're saying, but then perhaps the links to [cs.symfony.com](https://cs.symfony.com) need to be more prevalent, if that is the preferred way of reading the docs.

Reply:
> Maybe it's time to archive cs.symfony.com?
> _-- fabpot a few comments below_

If this happens, then what? Github becomes the main source of documentation? In that case, I'd like to point out this styling point again 😉

Reply:
Agreed, I'll try to find a workaround.